corehq/apps/locations/migrations/0013_get_stock_levels_from_config.py

```diff
@@ -0,0 +1,78 @@
+# encoding: utf-8
+from south.v2 import DataMigration
+from dimagi.utils.couch.database import get_db
+from corehq.apps.domain.models import Domain
+from corehq.apps.locations.models import LocationType
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        """call the save method on all relevant location types"""
+        db = get_db()
```

Comment:
should be `Domain.get_db()` (is equivalent but if we ever save these objects to a different database we won't have to clean this up later)